cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/Outlet.java

```diff
@@ -59,5 +94,15 @@ public Builder withLocation(Location location) {
             this.location = location;
             return this;
         }
-    }
+
+        public Builder withCompoundOutletRecords(List<CompoundOutletRecord> compoundOutletRecords) {
+            this.compoundOutletRecords = compoundOutletRecords;
+            return this;
+        }
+
+        public Builder withRatingGroupId(String ratingGroupId) {
+            this.ratingGroupId = ratingGroupId;
+            return this;
+        }
+	}
```

Comment:
Something is indented at the wrong level.

Reply:
woops!  fixed that.